src/test/util/blockfilter.cpp

```diff
@@ -11,6 +11,8 @@
 
 bool ComputeFilter(BlockFilterType filter_type, const CBlockIndex* block_index, BlockFilter& filter)
 {
+    LOCK(::cs_main);
```

Comment:
I am just curious - why prefix with `::`? There is no conflict with a local symbol under the same name, so the `::` shouldn't be necessary? Right now in the codebase there are 154 occurrences of `::cs_main` and 421 occurrences of `cs_main` (without the prefix).

Reply:
I do prefer specify the global namespace for the sake being explicit.

Reply:
Right, my understanding from review feedback is that it is preferred for new or updated code.

Reply:
For all global variables or just for `cs_main`? Global variables are already supposed to start with `g_`, so prefixing them with `::` is kind of redundant if the purpose is to show to the reader that this is a global variable.

Reply:
`cs_main` doesn't start with `g_`